e2e/e2e_central_traits_test.go

```diff
@@ -0,0 +1,102 @@
+package e2e
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/admin/private"
+	"github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
+	fmImpl "github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager/impl"
+)
+
+var _ = Describe("central traits", Ordered, func() {
```

Comment:
nit: let's align with `e2e_test.go

```suggestion
var _ = Describe("Central traits", Ordered, func() {
```